src/components/molecules/LoadingPage/LoadingPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo, useState } from "react";
+
+import { CustomLoader } from "types/CustomLoader";
+
+import { useCustomLoaders } from "hooks/useCustomLoaders";
+
+import "./loading.scss";
+
+const getRandomInt = (max: number, randomValue: () => number = Math.random) => {
+  return Math.floor(randomValue() * Math.floor(max));
+};
+
+// TODO: move this into useCustomLoaders, or settings, or similar?
```

Comment:
[TODO]

```typescript
// TODO: move this into useCustomLoaders, or settings, or similar?
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1240/commits/bda3c501e7d6cb66147eaba680698a87f81c41d1